tests/unit/makefile

```diff
@@ -27,27 +27,39 @@ GTEST_HEADERS = $(GTEST_DIR)/include/gtest/*.h \
 
 include $(BOUT_TOP)/make.config
 
+# Existence of this file indicates that GTEST has been downloaded
+GTEST_SENTINEL = $(BOUT_TOP)/googletest/README.md
+
 # House-keeping build targets.
 
 clean ::
 	@$(RM) -f gtest.a bout_test_main.a *.o $(TEST_OBJECTS)
 
 # Builds gtest.a and bout_test_main.a.
 
-# Usually you shouldn't tweak such internal variables, indicated by a
-# trailing _.
-GTEST_SRCS_ = $(GTEST_DIR)/src/*.cc $(GTEST_DIR)/src/*.h $(GTEST_HEADERS)
+# Download Google Test
+$(GTEST_SENTINEL) :
+	echo "Downloading Google Test"
+	curdir=$(PWD)
+	cd $(BOUT_TOP)/googletest
```

Comment:
I don't think you have to cd as the `submodule` command will update all submodules. Indeed I'm not sure if the `cd` does actually have any impact as each line in the recipe is executed in its own shell (as far as I understand).

Reply:
Good point, now removed.